kuma/core/decorators.py

```diff
@@ -15,6 +15,35 @@
 from .urlresolvers import reverse
 
 
+def add_shared_cache_control(response, **kwargs):
```

Comment:
How about ``kuma/core/cache.py`` or ``kuma/core/utils.py`` for this function?

Reply:
I'll go with `kuma/core/utils.py` if you don't mind.